pkg/vm/pipeline/pipeline.go

```diff
@@ -79,6 +87,47 @@ func (p *Pipeline) Run(r engine.Reader, proc *process.Process) (bool, error) {
 	}
 }
 
+func (p *Pipeline) Run2(proc *process.Process) (bool, error) {
+	var err error
```

Comment:
Please make the load a similar strategy to valuescan

Reply:
ok